gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -462,6 +472,22 @@ struct _GumBackpatch
   };
 };
 
+extern _Unwind_Reason_Code __gxx_personality_v0 (int version,
```

Comment:
I guess all of this should be behind:

```C
#if defined (HAVE_LINUX) && !defined (HAVE_ANDROID)
```

Like in the x86 backend?

Reply:
Nope... I tested on aarch64 android.